db/data/autotest_files/custom/specs.json

```diff
@@ -4,14 +4,11 @@
       "tester_type": "custom",
       "test_data": [
         {
-          "script_files": [
-            "autotest_01.sh"
-          ],
-          "category": [
-            "admin"
-          ],
+          "script_files": ["autotest_01.sh"],
```

Comment:
Please revert the formatting changes (since they aren't logically part of this PR)